umbraco-cloud/set-up/project-settings/availability-performance.md

```diff
@@ -0,0 +1,107 @@
+# Availability & Performance
+
+
+The "Availability & Performance" feature lets you see get an overview of your cloud projects' past and current health. Leveraging Azure metrics data, the page provides users with valuable insights into the availability and performance of the cloud project enabling them to monitor and address any issues that may impact the user experience.
+
+## Overview
+
+Under Availability & Performance, you'll find visualization and statistics for four sections:
```

Comment:
@sajumb There is only listed three sections below, is it then correct to say four? 🤔

Reply:
2Yeah, it was changed from four to three sections. It should state "three" here.